index.html

```diff
@@ -150,5 +150,36 @@ <h3 class="framework-text">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </section>
+        <section>
+            <p class="text-form">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form>
+                <label for = "name"></label>
+                <input type="text" id="name" class="input-padron" required placeholder="Full name">
+                <hr>
+                
+                <label for="mail"></label>
+                <input type="email" id="mail" class="input-padron" required placeholder="email address">
+                <hr>
+    
+                <label for="msj"></label>
+                <textarea rows="4" cols="50" maxlength="500" id="msj" class="input-padron" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Good job adding the contact form that sticks to the Figma design  :smile: , however as you will notice with me the `textarea` is not aligned as it should according to the Figma design, I think it could be better if you increase its width with the width of the email and the full name input :+1: 

![Screenshot from 2023-02-24 09-10-03](https://user-images.githubusercontent.com/57408419/221116194-2ed1a032-c8cf-4982-9c81-6a60c146c657.png)